contessa/models.py

```diff
@@ -174,7 +239,7 @@ def clsname(self):
         return camel_case[0].title() + camel_case[1:]
 
 
-def create_default_quality_check_class(result_table: ResultTable):
+def create_default_check_class(result_table: ResultTable, check_type: str = "quality"):
```

Comment:
What do you think about making this required ? I'd like to make it more explicit as this acts as factory. Also, in the future there are possible splits to accuracy, conformity etc.